install_tmp_&_tmp_scripts/main.py

```diff
@@ -0,0 +1,30 @@
+import time
+import datetime
+import json
+import sqlite3
+
+with open('config.json') as conf_f:
+    config = json.load(conf_f)
+
+db = sqlite3.connect(config['tmp_bd'])
+sql = db.cursor()
+sql.execute("""
+            CREATE TABLE IF NOT EXISTS tmp(
```

Comment:
```suggestion
            CREATE TABLE IF NOT EXISTS tmp(
```
tmp - плохое сокращение. Обычно  применяется для слова temp. В данном случае идет речь о логировании температуры. Предлагаю так и назвать TEMPERATURE_MONITORING или TEMPERATURE_LOG